rpm/0006-linux-user-Disable-safe_syscall-functionality-for-sb.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Frajo Haider <f_haider@gmx.at>
+Date: Thu, 12 Jan 2023 01:22:33 +0200
+Subject: [PATCH 6/6] linux-user: Disable safe_syscall functionality for sb2
+
+---
+ linux-user/qemu.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/linux-user/qemu.h b/linux-user/qemu.h
+index 5c964389c101ce00fcffe206bc69fb8b571634bc..15ff74749f4372269fff62cd98c1bf756cd84b2b 100644
+--- a/linux-user/qemu.h
++++ b/linux-user/qemu.h
+@@ -1,7 +1,7 @@
+ #ifndef QEMU_H
+ #define QEMU_H
+ 
+-#include "hostdep.h"
++// #include "hostdep.h"
```

Comment:
Is this required?

Reply:
In case it is just remove that line. I would make sense to explain this better as this change applies to any use of qemu since it not only applies to sb2.

Reply:
Yes it's required, because it'll redirect the syscall to an assembly implementation otherwise and sb2 can't handle that. The explanation is in the spec file.

Reply:
Generally speaking, I'm a firm believer of the "there shouldn't be commented code" rule. But I think this just might be the exception. But I also think Thaodan has a point - not everyone is going to go read the spec file to find the explanation here. Replacing this with `// #include "hostdep.h" commented out because sb2 handles syscalls going through c interface"` or something like that would make it clear. Bit this is all just nitpicking really.